content/projects/data-science-specific/bootcamp-exploratory-data-analysis/_index.md

```diff
@@ -40,30 +40,40 @@ It's good practice to structure your files well, so we'll expect you to have a s
 └──.gitignore 
 ```
 
+## Dealing with missing values in a dataset
+
+Before we dive into the tasks of this project, let's talk a bit about missing values. For a variety of reasons, the data we work with could have missing or invalid data. Depending on the kind of data we are working with and the extent of the data missing, we typically deal with missing data either by dropping it (for example in the case of entirely/nearly entirely missing columns or rows) or by replacing them with a suitable values.
+
+Later in this program you'll be learning how to train machine learning models using various kinds of data. When doing so, missing values cannot simply be ignored. Further, when it comes to datasets with many columns (also known as features), it is bad practice to drop an entire row for a few missing values. 
+
+Here is a resource for dealing with missing values : https://www.analyticsvidhya.com/blog/2021/10/a-complete-guide-to-dealing-with-missing-values-in-python/
+
 
 ## Questions
 
 The [data set](data.csv) consists of health and demographic data for the period 2014-2015, obtained from [Global Health Observatory Data Repository](http://apps.who.int/gho/data/node.main). Here is some [metadata](data-info.txt) that may be useful.
 
-1. Are there any missing values in the data set? You can deal with missing values by either dropping them or filling them with some value (in a variety of ways) or ignoring them. What do you think is best in this case? And why? Here is a resource for dealing with missing values : https://www.analyticsvidhya.com/blog/2021/10/a-complete-guide-to-dealing-with-missing-values-in-python/
+1. Load the dataset into a pandas DataFrame called `health_and_demographics_df`,  display the first 10 rows and determine how many missing values there are per feature. 
```

Comment:
Do we really need them to display the first 10 rows? 
When they determine how many missing values there are per feature then should they do anything with that number? Should it be in a variable or something? 

Is the purpose of this instruction to get the learner to notice that there are missing values, and then they can move on once they have noticed that? Why not just deal with all the missing values cohesively in step 2?

Reply:
This is the first data science project and we are trying to set them up for standard practice - after loading a dataset it's good to check that it's displaying normally - sometimes a person needs to specify a special delimiter character in the import function, or it doesn't actually split out the columns, etc. It is also helpful for someone reviewing the notebook (not meaning the code reviewer, but for whichever end user to see the basic structure of the data and features, so that they then understand the flow of work that follows. 

We need to see that they have addressed the missing values so not sure if we need them to store them as a variable up front - will ponder a bit more